### PR TITLE
Multisrc/madara

### PR DESCRIPTION
**Pull request by nyagami**
- closes https://github.com/LNReader/lnreader-plugins/issues/840
- closes https://github.com/LNReader/lnreader-plugins/issues/762
- closes https://github.com/LNReader/lnreader-plugins/issues/620
- closes https://github.com/LNReader/lnreader-plugins/issues/618
- closes https://github.com/LNReader/lnreader-plugins/issues/516
- closes https://github.com/LNReader/lnreader-plugins/issues/498
- closes https://github.com/LNReader/lnreader-plugins/issues/461
- closes https://github.com/LNReader/lnreader-plugins/issues/350
- closes https://github.com/LNReader/lnreader-plugins/issues/864

### CODE REVIEW

File: scripts/multisrc/madara/sources.json [hunk 1]
```diff
@@ -368,5 +368,85 @@
...
+  },
+  {
+    "id": "dragontea",
+    "sourceSite": "https://dragontea.ink/",
+    "sourceName": "Dragon Tea",
+    "options": {
+      "useNewChapterEndpoint": true,
+      "lang": "English"
+    }
+  },
```
Comment (error7404) [inline scripts/multisrc/madara/sources.json:398]: Dragon tea doesn't work without changing some stuff in the generator, I was working on it

Comment (nyagami) [reply scripts/multisrc/madara/sources.json:398]: lmao, they are mangas

Comment (nyagami) [reply scripts/multisrc/madara/sources.json:398]: I think we should have path options for some sites like this. Novels still work fine

Screenshot_20240429-155446

Comment (nyagami) [reply scripts/multisrc/madara/sources.json:398]: oh, it doesnt, what is this language XD

**Pull request merged**